src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/Sql/PaymentsSqlClient.cs

```diff
@@ -39,3 +39,16 @@ public PaymentsSqlClient()
         _sqlServerClient = SqlServerClientProvider.GetSqlServerClient(connectionString);
     }
 }
+
+public static class PaymentsSqlClientExtensions
+{
+    public static List<TestSupport.Payments> GetPayments(this PaymentsSqlClient paymentsSqlClient, ScenarioContext context)
+    {
+        var paymentModel = paymentsSqlClient.GetPaymentsModel(context);
+
+        if (paymentModel == null)
+            Assert.Fail("Payments model is null");
```

Comment:
This will never be null because there is a .Single call in the GetPaymentsModel method.